crates/llm-chain-sagemaker-endpoint/src/executor.rs

```diff
@@ -2,6 +2,7 @@ use crate::model::Formatter;
 use crate::model::Model;
 use async_trait::async_trait;
 
+use llm_chain::options;
 use llm_chain::options::Opt;
 use llm_chain::options::Options;
 use llm_chain::options::OptionsCascade;
```

Comment:
```suggestion
use llm_chain::options::{self, Opt, Options, OptionsCascade};
```

Reply:
the `use llm_chain::options` is for getting the top-level `options!` macro. For the rest (`Opt`, `Options`, `OptionsCascade`) I put them into one line.

Or would you prefer
```
use llm_chain::{options, options:{Opt, Options, OptionsCascade}};
```
This seems overly complicated for me.

Reply:
Ah, I didn't realize it was a macro sharing a name with a module, interesting. Your way is fine.